### ISSUE TITLE
more project / workspace operations on dashboard

### ISSUE THREAD

**Issue by alexweininger** [Kruemelkatze/vscode-dashboard#3]
I think it'd be great if there were a few more functions available to the user when he is confronted with their dashboard.

some of those functionalities could be:

- open in a new window
- edit the project name from the dashboard
- remove from the dashboard
- go to workspace settings for that workspace
- go to github repo of that repository
- reorder tiles by dragging them

**Comment by Kruemelkatze** [comment]
These are some neat ideas! 

> open in a new window

Already works by CTRL +Click. Maybe should have mentioned that in the readme :)


> go to workspace settings for that workspace
> go to github repo of that repository

These are interesting features! I need to have a look on the logic behind that, though. Firstly, if the VSCode API supports opening the workspace settings of another workspace. Secondly, how the extension should check for the Github URL. And as I think about it: Some people (including me and coworkers) use this extension mostly/also for work, where the projects are hardly pushed to GitHub. This may be an optional option for the project, however I would like to have the user select only the minimum amount of information when creating a project.


> edit the project name from the dashboard
> remove from the dashboard
> reorder tiles by dragging them

I initially planned to make a designated editing mode for exactly these functions. I decided against it as I did not want to include too many custom UI elements into VSCode (which is also discouraged by the docs). But if it's a desired feature for many users, we could surely implement this! So +1 this if you would like such a feature!!! ;)
